pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -386,6 +387,19 @@ def _do_catch(self, pokemon, encounter_id, catch_rate_by_ball, is_vip=False):
                         'exp': sum(response_dict['responses']['CATCH_POKEMON']['capture_award']['xp'])
                     }
                 )
+
+                # We could refresh here too, but adding 3 saves a inventory request
+                candy = inventory.candies().get(pokemon.num)
```

Comment:
Does this mean that there would be a separate count for pidgey candy vs pidgeotto candy (as an example)?